index.js

```diff
@@ -693,46 +745,77 @@ const autoScript = {
-    exec('zip dist/timezones.geojson.zip dist/combined.json', cb)
+    if (argv.skip_zip) {
+      overallProgress.beginTask('Skipping zip')
+    } else {
+      overallProgress.beginTask('Zipping geojson')
+      let zipFile = argv.dist_dir + '/timezones.geojson.zip'
+      let jsonFile = argv.dist_dir + '/combined.json'
+      exec('zip ' + zipFile + ' ' + jsonFile, cb)
+    }
   }],
   zipGeoJsonWithOceans: ['mergeZones', function (results, cb) {
-    overallProgress.beginTask('Zipping geojson with oceans')
-    exec('zip dist/timezones-with-oceans.geojson.zip dist/combined-with-oceans.json', cb)
+    if (argv.skip_zip) {
+      overallProgress.beginTask('Skipping with oceans zip')
+    } else {
+      overallProgress.beginTask('Zipping geojson with oceans')
+      let zipFile = argv.dist_dir + '/timezones-with-oceans.geojson.zip'
+      let jsonFile = argv.dist_dir + '/combined-with-oceans.json'
+      exec('zip ' + zipFile + ' ' + jsonFile, cb)
+    }
   }],
   makeShapefile: ['mergeZones', function (results, cb) {
-    overallProgress.beginTask('Converting from geojson to shapefile')
-    rimraf.sync('dist/combined-shapefile.*')
-    exec(
-      'ogr2ogr -f "ESRI Shapefile" dist/combined-shapefile.shp dist/combined.json',
-      function (err, stdout, stderr) {
-        if (err) { return cb(err) }
-        exec('zip dist/timezones.shapefile.zip dist/combined-shapefile.*', cb)
-      }
-    )
+    if (argv.skip_shapefile) {
+      overallProgress.beginTask('Skipping shapefile creation')
+    } else {
+      overallProgress.beginTask('Converting from geojson to shapefile')
+      let shapeFileGlob = argv.dist_dir + '/combined-shapefile.*'
+      rimraf.sync(shapeFileGlob)
+      let shapeFile = argv.dist_dir + '/combined-shapefile.shp'
+      let jsonFile = argv.dist_dir + '/combined.json'
+      exec(
+        'ogr2ogr -f "ESRI Shapefile" ' + shapeFile + ' ' + jsonFile,
+        function (err, stdout, stderr) {
+          if (err) { return cb(err) }
+          let shapeFileZip = argv.dist_dir + '/timezones.shapefile.zip'
+          exec('zip ' + shapeFileZip + ' ' + shapeFileGlob, cb)
+        }
+      )
+    }
   }],
   makeShapefileWithOceans: ['mergeZones', function (results, cb) {
-    overallProgress.beginTask('Converting from geojson with oceans to shapefile')
-    rimraf.sync('dist/combined-shapefile-with-oceans.*')
-    exec(
-      'ogr2ogr -f "ESRI Shapefile" dist/combined-shapefile-with-oceans.shp dist/combined-with-oceans.json',
-      function (err, stdout, stderr) {
-        if (err) { return cb(err) }
-        exec('zip dist/timezones-with-oceans.shapefile.zip dist/combined-shapefile-with-oceans.*', cb)
-      }
-    )
+    if (argv.skip_shapefile) {
+      overallProgress.beginTask('Skipping with oceans shapefile creation')
+    } else {
+      overallProgress.beginTask('Converting from geojson with oceans to shapefile')
+      let shapeFileGlob = argv.dist_dir + '/combined-shapefile-with-oceans.*'
```

Comment:
The callback `cb` needs to be called in this block in order to indicate that this task has completed successfully. Also, one small code style guide is that it could be cleaner to include a return statement in this block and then the rest of the method won't be so indented as much. ie:

```js
if (argv.skip_shapefile) {
  overallProgress.beginTask('Skipping with oceans shapefile creation')
  return cb()
}
overallProgress.beginTask('Converting from geojson with oceans to shapefile')
...
```

Reply:
Done in commit 5

Reply:
Done in commit 5